applications/asset_tracker_v2/tests/gps_module/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  asset_tracker_v2.gps_module_test.tester:
+    platform_allow: native_posix qemu_cortex_m3 nrf9160dk_nrf9160ns
```

Comment:
If we expect merge after the current upmerge completes, we should change the last `platform_allow` to the new format `nrf9160dk_nrf9160_ns`

Reply:
Good to know. I am not sure when this will be merged. I will keep this in mind

Reply:
@jorgenmk  I have updated the board name now.